src/controllers/confirmation.controller.ts

```diff
@@ -0,0 +1,67 @@
+import { NextFunction, Request, Response } from "express";
+import {formatDateForDisplay} from "../client/date.formatter";
+import logger from "../logger";
+import {PTFCompanyProfile} from "../model/company.profile";
+import * as templatePaths from "../model/template.paths";
+import * as sessionService from "../services/session.service";
+import {ALREADY_SUBMITTED, COMPANY_NUMBER, COMPANY_PROFILE, SIGN_IN_INFO, USER_PROFILE} from "../session/keys";
+import {IUserProfile} from "../session/types";
+
+const ACCOUNTS_NEW_DEADLINE: number = 28;
```

Comment:
I can't see anything in the story https://companieshouse.atlassian.net/browse/LFA-1166 that relates to the numbers 14 and 28. What are these? Code should relate to a business requirement somewhere.

Reply:
Strictly speak no it is not related to 1166 but there is only one ticket for the controller and this will cover both the no longer and is still required alternatives. Given this we will need these values.

Reply:
But where's the evidence that it's 14 not 15 days? Your word against mine or developers' word against testers. I really think that we should request an analyst to at least add an addendum to the story in JIRA or a short comment clarifying this.

Reply:
This controller will change as we add to the project along with the TODO this is really just something to be going on with so that we can start the work for other tickets but we need to do so in order to bring many of thes changes about. This is in no way written in stone.